tests/console/validate_zfcp_multipath.pm

```diff
@@ -36,13 +36,20 @@ sub verify_plain_scsi_block_devices {
 sub verify_multipath_block_devices {
     my $mpath = script_output "multipath -l";
     my @mpath = split /\n/, $mpath;
+    my $origin_wwid = get_dynamic_wwid();
```

Comment:
there is no "origin" in the name, it doesn't change during one run.

Reply:
updated